Cython/Includes/libcpp/limits.pxd

```diff
@@ -0,0 +1,61 @@
+cdef extern from "limits" namespace "std" nogil:
+   enum float_round_style:
+        round_indeterminate       = -1
```

Comment:
These literal values are not needed; just omit them and we'll use the underlying C++ ones.